.drone.yml

```diff
@@ -22,6 +22,8 @@ data:
           value_from_secret: auction-ui/main.auction-ui.k8s.sidestream.tech/frontend/contact_email
         - key: STAGING_BANNER_URL
           value_from_secret: auction-ui/main.auction-ui.k8s.sidestream.tech/frontend/staging_banner_url
+        - key: PRODUCTION_BANNER_URL
+          value_from_secret: auction-ui/main.auction-ui.k8s.sidestream.tech/frontend/production_banner_url
```

Comment:
This will only add it to the `main.auction-ui.k8s.sidestream.tech` (which is staging). You also need to add it down to `publish-frontend-makerdao-on-tag` step